option optimize_for = LITE_RUNTIME;

message LegionWorshipAnswer
{
 required uint32 result = 1;
 required uint32 crit = 2;
}
